src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -1023,6 +1084,19 @@ internal static void EscapeAndAppendString(StringBuilder dest, string source)
             dest.Append('"');
         }
 
+        /// <summary>
+        /// Never escape output.
+        /// </summary>
+        internal static void AppendStringWithEscapeNever(StringBuilder dest, string source)
```

Comment:
This method isn't actually used.  I understand the original intent, but I think just calling `.Append()` rather than calling this method is preferred.  So you should remove this.

Reply:
Removed.